src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,18 @@
+import { Link } from 'react-router-dom';
+import { FC } from 'react';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: FC<Props> = ({ person }) => {
+  return (
+    <Link
+      to={person.slug}
+      className={person.sex === 'f' ? 'has-text-danger' : ''}
```

Comment:
Use the `classnames` library, it will improve your code

Reply:
Good point!